app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -250,6 +250,7 @@
   <dimen name="faq_single_padding_top">48dp</dimen>
   <dimen name="faq_single_padding_end">32dp</dimen>
   <dimen name="faq_single_padding_bottom">132dp</dimen>
+  <dimen name="faq_single_activity_scroll_padding_Bottom">0dp</dimen>
```

Comment:
Rename to `faq_single_activity_scroll_padding_bottom`

Reply:
This has also been renamed to |faq_single_activity_scroll_padding_bottom|  in dimens.xml sw600dp-land.